msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -181,16 +177,31 @@ For more information, see [roles in a Teams meeting](https://support.microsoft.c
 
 After you design your app based on participant roles in a meeting, you can identify each user type for meetings and select what they can access.
 
+In public channel meetings, team owners can manage meeting app permissions through team settings.​ Channel meeting app permissions refer to the team app CRUD model (not the non-channel meeting app CRUD model). If a user can add an app to a team or channel, they can use it in the meeting.​
```

Comment:
Could we explicitly clarify the relationship between apps installed to the team, and apps installed to a channel meeting? (If we call this out, I must have missed it.) The impression I get is that they're separate sets--installing an app to a team doesn't automatically make it available to a channel meeting, and vice versa. Is this correct? If so, I think this is an important point.

Reply:
clarified this in first page with type of supported meeting description note. please take a look